### PR TITLE
Fix broken reference in `docs/web_exceptions.rst`

### PR DESCRIPTION
**Pull request by l1storez**
## What do these changes do?

 This change corrects multiple unrendered intersphinx class reference in the `web_exceptions.rst` document.

## Are there changes in behavior for the user?

 No

## Related issue number

 #5518 

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: docs/web_exceptions.rst [hunk 1]
```diff
@@ -479,13 +479,13 @@ HTTP exceptions for status code in range 400-499, e.g. ``raise web.HTTPNotFound(
    :exc:`HTTPClientError`.
 
    :param link: A link to a resource with information for blocking reason,
-                :class:`str` or :class:`URL`
+                :class:`str` or :class:`yarl.URL`
```
Comment (webknjaz) [inline docs/web_exceptions.rst]:
```suggestion
                :class:`str` or :class:`~yarl.URL`
```

File: docs/web_exceptions.rst [hunk 2]
```diff
@@ -479,13 +479,13 @@ HTTP exceptions for status code in range 400-499, e.g. ``raise web.HTTPNotFound(
...
 
    For other parameters see :exc:`HTTPException` constructor.
 
    .. attribute:: link
 
-      A :class:`URL` link to a resource with information for blocking reason,
+      A :class:`yarl.URL` link to a resource with information for blocking reason,
```
Comment (webknjaz) [inline docs/web_exceptions.rst]:
```suggestion
      A :class:`~yarl.URL` link to a resource with information for blocking reason,
```

**Comment by l1storez** [comment]
fixed

**Review by webknjaz — APPROVED** [review]

**Comment by webknjaz** [comment]
Backport to v3.8 is unnecessary because the patched doc didn't exist back then.

**Pull request merged**